### PR TITLE
trezor: bump lib version, implement new passphrase-on-device UI

### LINKED ISSUE #1946: Trezor One error with Electron Cash 4.1.0 on Linux
The latest 4.1.0 with Cash Fusion doesn't work anymore on Linux with Trezor One hardware wallet with firmware 1.9.0.

When trying to sign a transaction Electron Cash shows the error:

_get_passphrase() got an unexpected keyword argument 'available_on_device'_

### PR DESCRIPTION
**Pull request by EchterAgo**
from https://github.com/spesmilo/electrum/commit/4cd50dd75a2613a3d631cefaf0bf88831786b6fd with adjustments

closes #1946

### CODE REVIEW

File: plugins/trezor/cmdline.py [hunk 1]
```diff
@@ -1,7 +1,22 @@
 from electroncash.plugins import hook
-from .trezor import TrezorPlugin
+from electroncash.i18n import _
+from electroncash.util import print_stderr
+from .trezor import TrezorPlugin, PASSPHRASE_ON_DEVICE
 from ..hw_wallet import CmdLineHandler
 
+class TrezorCmdLineHandler(CmdLineHandler):
+    def __init__(self):
+        self.passphrase_on_device = False
```
Comment (EchterAgo) [inline plugins/trezor/cmdline.py:9]: Do we want to enable this by default? Electrum has it set to false.

**Comment by EchterAgo** [comment]
https://ec.loping.net/4.1.0-24-g8ac98c673/

**Comment by EchterAgo** [comment]
Updated to include `trezor==0.12.1` in the deterministic requirements.

**Comment by EchterAgo** [comment]
https://ec.loping.net/4.1.0-24-g590e532f2/

File: plugins/trezor/cmdline.py (hunk 1 above)
Comment (cculianu) [reply plugins/trezor/cmdline.py:9]: I.. do not know. I personally hate passphrases... *shrug*

**Comment by cculianu** [comment]
LGTM

**Pull request merged**